test/shared-class.test.js

```diff
@@ -295,6 +295,70 @@ describe('SharedClass', function() {
       expect(methods).to.not.contain(INST_METHOD_NAME);
       expect(methods).to.not.contain('instTestMethod');
     });
+
+    it('excludes disabled prototype methods from the method list using ' +
+    'a method alias', function() {
+      var INST_METHOD_NAME = 'instTestMethod';
+      var INST_METHOD_ALIAS = 'instTestMethodAlias';
```

Comment:
while we're here, please extract these variables out after L#80 and put the initialization in `beforeEach()` block (same as L#251) so that there is no repetition  of same code.